game/views.py

```
@@ -625,64 +830,60 @@ def logout_view(request):
+
     try:
         deleted, resigned = cleanup_stale_games()
-        return JsonResponse({
-            'status': 'success',
-            'deleted_games': deleted,
-            'resigned_games': resigned
-        })
+        return JsonResponse({"status": "success",
+                             "deleted_games": deleted,
+                             "resigned_games": resigned})
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)